lib/rtp_vp9/vp9_payload_descriptor.ex

```diff
@@ -127,19 +127,19 @@ defmodule Membrane.RTP.VP9.PayloadDescriptor do
       when byte_size(rest) > 0 do
     <<i::1, _p::1, _l::1, f::1, _bevz::4>> = header
 
-    if i != f do
-      {:error, :malformed_data}
+    # If the I bit is set to zero, then F MUST also be set to zero
+    # Relationship between I and F bit can by described by logic formula
+    #         ~I => ~F which is equivalent to (I or ~F)
+    with true <- i == 1 or not (f == 1),
+         <<decoded_header>> <- header,
```

Comment:
this will always match, you can extract that before `with`

Reply:
This line is removed in the following PR-s anyway.

Reply:
hmm that means we have too many PRs :P